mslib/msui/mscolab.py

```diff
@@ -1027,9 +1027,16 @@ def add_operation(self):
             self.error_dialog.showMessage('Path can\'t contain spaces or special characters')
             return
 
+        user_template = MSUIDefaultConfig.new_flighttrack_template
+        xml_waypoints = "".join(
```

Comment:
This did not work. we need the coordinates, you can do this similiar to 

https://github.com/annapurna-gupta/MSS/blob/develop/mslib/msui/mscolab.py#L1162

you get by waypoints_model.get_xml_content() your needed xmlcontent.